app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -476,9 +479,7 @@ class LuciBuildService {
         ],
         'user_agent': const <String>['luci-scheduler'],
       },
-      properties: <String, String>{
-        'git_ref': commitSha,
-      },
+      properties: localProperties,
```

Comment:
Will this override the existing input properties? For example, are you required to copy all properties into the POST request, then add the cache clobber or clear build artifacts property?

Reply:
it will be merged and replace any existing ones: https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/model/luci/buildbucket.dart#L387

Reply:
Thanks for checking Keyong!